eth/peer.go

```diff
@@ -136,7 +136,7 @@ func (p *peer) SendTransactions(txs types.Transactions) error {
 	for _, tx := range txs {
 		p.knownTxs.Add(tx.Hash())
 	}
-	return p2p.SendEthSubproto(p.rw, TxMsg, txs, p.ID())
```

Comment:
I just reverted to what master branch had.
I think `send/SendItems` was split into those 2 functions so that we have better control over log messages.
Since we aren't logging those anymore, I just switched them back.